worlds/mountain_fog.sdf

```diff
@@ -0,0 +1,268 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sdf version="1.9">
+  <world name="default">
```

Comment:
`  <world name="mountain_fog">`

Reply:
Oh, yes, I missed that. That part was left while working on data.dem, it should be data.dt2. I corrected the error, but making complex shapes collisions no longer seems logical. Instead, simpler collisions can be used